src/components/Avatar/Avatar.jsx

```diff
@@ -1,33 +1,20 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-
-import AvatarBase from './AvatarBase'
+import classNames from 'classnames'
 import { Camera } from 'react-feather'
 
-import './Avatar.css'
-
-const Replace = ({ size, image, onClick }) =>
-  <AvatarBase upload src={image} width={size} height={size} onClick={onClick} />
+import { Typography } from '../'
 
-Replace.propTypes = {
-  size: PropTypes.number,
-  image: PropTypes.string.isRequired,
-  onClick: PropTypes.func
-}
+import styles from './Avatar.module.scss'
 
-const NewUpload = ({ style, onClick }) => (
-  <Camera
-    className='Avatar upload'
-    color='var(--grey-200)'
-    style={style}
-    onClick={onClick}
-  />
-)
-
-NewUpload.propTypes = {
-  style: PropTypes.object,
-  onClick: PropTypes.func
-}
+const COLORS = [
+  styles.colorPrimary,
+  styles.colorSuccess,
+  styles.colorDanger,
+  styles.colorNeutral,
+  styles.colorWarning,
+  styles.colorCyan
+]
```

Comment:
I think the colors used should be redefined with different names in colors.scss.
It seems strange to use colors declared as colorSuccess or colorDanger.